frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -230,15 +225,27 @@ end
 
 function FileSearcher:showSearchResultsMessage(no_results)
     local text = no_results and T(_("No results for '%1'."), self.search_value)
-    if self.no_metadata_count ~= 0 then
-        local txt = T(N_("1 book has not been searched in.", "%1 books has not been searched in.",
+    if self.no_metadata_count == 0 then
+        UIManager:show(InfoMessage:new{
+            text = text,
+        })
+    else
+        local txt = T(N_("1 book has been skipped.", "%1 books have been skipped.",
             self.no_metadata_count), self.no_metadata_count) .. "\n" ..
-            _("Please do 'Extract and cache book information' from Plus menu to search in all books metadata.")
+            _("No book metadata extracted yet.\nExtract metadata now?")
```

Comment:
It's not really "No book metadata...", rather "Some book metadata...", "Not all book metadata...".
I think we could be more verbose here ;)

Reply:
As a continuation of the first sentence, we are talking about "skipped books".

Reply:
I tried to avoid one more single/plural choice.

Reply:
"not all..." still avoids that, and I also think that'd sound a bit better.